pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentSubscription.java

```diff
@@ -394,8 +416,16 @@ public void acknowledgeMessage(List<Position> positions, AckType ackType, Map<St
                 }
             }
         }
-
+        TopicName topicName1 = TopicName.get(getTopicName());
+        Optional<Policies> policies = null;
+        try {
+            policies = topic.getBrokerService().pulsar().getConfigurationCache().policiesCache()
+                    .get(AdminResource.path(POLICIES, topicName1.getNamespaceObject().toString()));
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
```

Comment:
Should we abstract it as a method?

Reply:
There are one or two places used in a class, and there is no meeting point in itself.Is it necessary to abstract it into a method？